src/rosdiscover/observer/observer.py

```diff
@@ -23,7 +23,23 @@ def for_container(cls,
                       container: str,
                       config: 'Config',
                       ) -> Iterator['Observer']:
-        """Constructs and interpreter for a given running container"""
+        """Constructs and interpreter for a given running container
+
+        Parameters
+        ----------
+        container: str
+            The image id or name of a container running a ROS system
+        config: Config
+            The configuration information that gives information about how to set up the
```

Comment:
It might be worth updating this to reflect that the `Config` should describe the configuration used by the container that is running.

Reply:
Done